### DISCUSSION TITLE
Add Relay Market Place

### DISCUSSION THREAD

**Discussion by tkhumush** [damus-io/damus#1258] [Ideas]
Paid Relays need a way and place to advertise their value propositions.
Creating a Relay Market Place in Damus could help:
1) Relays advertise and generate some revenue.
2) Damus can also make some "ad/sale Revenue".

**Comment by jclmenezes** [comment]
I went to the relays website and tried to add several, but when I click on the add relay option, the button doesn't work, any suggestions for a solution?
